src/components/album/Album.js

```diff
@@ -30,15 +31,41 @@ const AlbumTitle = styled('h3')({
 
 function Album({ album, wishListId }) {
   const [openDialog, setOpenDialog] = useState(false);
+  const [reviewsUpdated, setReviewsUpdated] = useState(false);
+  const [fullWidth, setFullWidth] = React.useState(true);
+  const [maxWidth, setMaxWidth] = React.useState('sm');
 
   const handleOpenDialog = () => {
     setOpenDialog(true);
   };
 
   const handleCloseDialog = () => {
     setOpenDialog(false);
+    // Reset the reviewsUpdated state when the dialog is closed
+    setReviewsUpdated(false);
   };
 
+  const handleMaxWidthChange = (event) => {
+    setMaxWidth(event.target.value);
+  };
+
+  const handleFullWidthChange = (event) => {
+    setFullWidth(event.target.checked);
+  };
+
+  // Function to refresh reviews
+  const refreshReviews = () => {
+    // Set reviewsUpdated to true to trigger a re-render of AlbumReviews
```

Comment:
I'm not understanding how this re-renders the album reviews?

Reply:
@akosasante oh that task was not fiished.. I wasn't sure how to do it without useState and useEffect hooks